### DISCUSSION TITLE
The non-metro language flyout is still there in Win10 and Win11

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#3182] [Ideas]
It is a part of Language Bar. Can EP make some hooks so to make it appear instead of the default XAML language flyout even when no language bar is used?
![изображение](https://github.com/valinet/ExplorerPatcher/assets/1271736/36a25626-6a59-445b-87ac-33bfab0213bb)

I think, it is in msutb.dll.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

As EP is tested against the current release of windows 10 and 11 with the current monthly updates.
